Proposal.md

```diff
@@ -1,3 +1,21 @@
+# Section 2: Description of Data
+
+In this project , we are using the **Cost of Living - NUMBEO Dataset** which is avaialable in [Kaggle](https://www.kaggle.com/joeypp/cost-of-living-numbeo-dataset) .This data set contains cost of living information of 160 cities across the globe with variety of information such as country, city name, region, rent, cost of owning a house, car etc.
+The Kaggle Dataset has been extracted from [numbeo](https://www.numbeo.com/cost-of-living) which is the world’s largest crowd-sourced cost of living database which collects statitics on the quality of life informations including housing indicators, crime rates, and healthcare. The website contains around 7,541,898 prices in 10,529 cities which have been entered by 630,509 contributors.
+We will be visualizing the data of 160 cities which was collected in the year 2020.The dataset has 56 different columns that contain cost of living
+information which we have processed and broadly classified into major categories such as `monthly salary`, `rent`, `grocery`, `utility`, `shopping`, `entertainment`, `fitness`, `property_price`, `transportation`, `childcare expenses`,`monthly_savings` miscellaneous for ease of visualization and understanding.Although the original dataset contained all the prices in Euro, we have converted the prices to USD.
+Our app will allow users to view these categorization which can be easily filtered to view the statistics for different cities or region or country. Since, the main aim of this project is to give an overview of the cost of living standards to new Graduates, we have processed the data based on following assumptions:
```

Comment:
```suggestion
Our app will allow users to view these categorization which can be easily compared with other cities or regions. Since, the main aim of this project is to give an overview of the cost of living standards to new Graduates, we have processed the data based on following assumptions:
```